src/pdf_beamtime/README.md

```diff
@@ -0,0 +1,33 @@
+## Functionality
+
+This package has two nodes. 
+
+The first node is pdf_beam_env inside pdf_beam_env.cpp. This nodes invokes two service calls to create a new obstacle and change the parameters of an existing obstacle. 
```

Comment:
Do we need this node at all? It is a sensible test, but I don't know if it belongs in the package...

Reply:
Removed the node pdf_beam_env.